pkg/fftypes/operation.go

```diff
@@ -22,16 +22,22 @@ package fftypes
 type OpType = FFEnum
 
 var (
-	// OpTypeBlockchainBatchPin is a blockchain transaction to pin a batch
-	OpTypeBlockchainBatchPin OpType = ffEnum("optype", "blockchain_batch_pin")
+	// OpTypeBlockchainPinBatch is a blockchain transaction to pin a batch
+	OpTypeBlockchainPinBatch OpType = ffEnum("optype", "blockchain_pin_batch")
```

Comment:
Should make sure @eberger727 is plugged into these name changes for any filtering in-flight on the firefly-ui